easybuild/easyblocks/b/boost.py

```diff
@@ -129,6 +135,9 @@ def build_step(self):
 
         bjamoptions = " --prefix=%s" % self.objdir
 
+        if self.cfg['parallel']:
+            bjamoptions += " -j%s" % self.cfg['parallel']
```

Comment:
same here, use `self.cfg['buildopts'].update(...)` instead of defining/updating/using `bjamoptions`?